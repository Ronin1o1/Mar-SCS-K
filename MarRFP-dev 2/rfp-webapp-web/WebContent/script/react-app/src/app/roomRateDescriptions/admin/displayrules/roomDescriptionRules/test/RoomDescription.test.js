import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import RoomDescription from "../content/RoomDescription";

describe("RoomDescription", () => {
  
    it("renders without crashing", () => 
    {    mount( <RoomDescription / > );
    });
    {/* it("renders correctly", () => 
    {    const tree = mount( <RoomDescription / > );        
    expect(toJson(tree)).toMatchSnapshot();    
    });
     */}
    
});
