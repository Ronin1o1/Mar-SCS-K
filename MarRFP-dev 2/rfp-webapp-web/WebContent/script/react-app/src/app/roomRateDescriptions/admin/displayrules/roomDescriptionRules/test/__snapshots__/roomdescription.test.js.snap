// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoomDescription rendersÂ correctly 1`] = `
<RoomDescription>
  <RoomDescriptionContextProvider>
    <form
      id="thisForm"
      name="thisForm"
    >
      <table
        className="section"
      >
        <tbody>
          <tr>
            <td
              className="header"
            >
              Room / Rate Description -  Administration - Display Rules : Room Description - Channels and Entries
            </td>
          </tr>
          <tr
            className="headerHR"
          >
            <td
              valign="top"
            />
          </tr>
        </tbody>
      </table>
      <table>
        <tbody>
          <tr>
            <td>
              <table
                className="zero_Height"
              >
                <tbody>
                  <tr>
                    <td
                      className="field_Name"
                    >
                      Channel:
                      <select
                        onChange={[Function]}
                      >
                        <option
                          value="show"
                        >
                          Please select a channel
                        </option>
                        <option
                          key="[object Object]"
                          value="null_undefined_null"
                        />
                      </select>
                      <div
                        style={
                          Object {
                            "marginLeft": "15px",
                            "marginTop": "10px",
                          }
                        }
                      />
                    </td>
                  </tr>
                </tbody>
              </table>
            </td>
          </tr>
        </tbody>
      </table>
    </form>
  </RoomDescriptionContextProvider>
</RoomDescription>
`;
